src/flow/ext_multiroute_flow.jl

```diff
@@ -181,7 +181,7 @@ function slope{T<:AbstractFloat}(
   )
   slope = 0
   for e in edges(flow_graph)
-    if cut[dst(e)] - cut[src(e)] > 0 &&
+    if cut[dst(e)] == 2 > cut[src(e)] &&
```

Comment:
OH - one other thing - this fixes something that you ran into. Could you add a test for this so that we've got a check for it in the future? (Sorry I didn't mention this before.)